backend/src/test/java/capstone/myfinancemanager/manager/service/TransactionServiceTest.java

```diff
@@ -116,5 +138,18 @@ void deleteTransactionNotExistsTest() {
         verify(transactionRepo, times(0)).deleteById(transactionDto1.getId());
     }
 
+    @Test
+    void updateTransactionTest() {
+
+        //when
+        when(randomUUIDGenerator.getRandomId()).thenReturn(randomTestId);
+        when(timestampService.now()).thenReturn(Instant.parse("2022-08-23T09:22:41.255023Z"));
+        when(transactionRepo.save(any())).thenReturn(transaction1);
+        when(transactionRepo.findById(transaction1.getId())).thenReturn(Optional.ofNullable(transaction1));
+
+        Transaction acutalTansaction = transactionService.updateTransaction(transaction1.getId(), transactionCreationDto1);
+
+        Assertions.assertEquals(transaction1.getId(), acutalTansaction.getId());
```

Comment:
This only checks that for id equality but not that the update was applied. Maybe compare the objects or verify that the repos save method was called with the correct object?